src/components/subscriptions/LicenseStatus.jsx

```diff
@@ -43,5 +44,6 @@ export default function LicenseStatus({ user }) {
 LicenseStatus.propTypes = {
   user: PropTypes.shape({
     licenseStatus: PropTypes.string.isRequired,
+    pendingSince: PropTypes.string.isRequired,
```

Comment:
I might recommend making this a non-required prop and render the "pending since" date conditionally, just in case the API response includes `null` on the `last_reminder_date`.